build.rs

```diff
@@ -21,7 +21,7 @@ BACKTRACE_RS_ANDROID_APIVERSION __ANDROID_API__
 fn build_android() {
     // Create `android-api.c` on demand.
     // Required to support calling this from the `std` build script.
-    let out_dir = env::var_os("OUT_DIR").unwrap();
+    let out_dir = std::fs::canonicalize(env::var_os("OUT_DIR").unwrap()).unwrap();
```

Comment:
Canonicalization can fail. Maybe use the existing `OUT_DIR` unchanged if canonicalization fails?

Reply:
Thank you for the quick review, I added a commit to use `OUT_DIR` as is if canonicalization fails.